src/tir/transforms/make_packed_api.cc

```diff
@@ -120,6 +120,11 @@ PrimFunc MakePackedAPI(PrimFunc&& func, int num_unpacked_args) {
   int num_args = static_cast<int>(func_ptr->params.size());
   ICHECK_LE(num_unpacked_args, num_args);
 
+  bool pack_args = true;
+  if (num_unpacked_args == -1) {
+    pack_args = false;
+    num_unpacked_args = 0;
+  }
   int num_packed_args = num_args - num_unpacked_args;
```

Comment:
seems like we could `ICHECK_GE(num_unpacked_args, 0)` here now

Reply:
Updated